Sources/StreamChatUI/Appearance+Images.swift

```diff
@@ -59,6 +59,21 @@ public extension Appearance {
         public var emptyChannelListMessageBubble: UIImage = loadImageSafely(with: "empty_channel_list_message_bubble")
         public var download: UIImage = loadSafely(systemName: "icloud.and.arrow.down", assetsFallback: "download")
 
+        // MARK: - Recording
+
+        public var mic: UIImage = loadSafely(systemName: "mic", assetsFallback: "mic")
```

Comment:
Maybe it's best if you update this page in this PR, so we don't forget it: https://getstream.io/chat/docs/sdk/ios/common-content/reference-docs/stream-chat-ui/appearance.images/.